packages/user/src/supertokens/change-password.ts

```diff
@@ -0,0 +1,36 @@
+import client from "@/api/axios";
+import { toast } from "react-toastify";
+
+const changePassword = async (
+  oldPassword: string,
+  newPassword: string,
+  apiBaseUrl: string
+): Promise<boolean | undefined> => {
+  let success = false;
+
+  try {
+    const response = await client(apiBaseUrl).post(
+      "/change_password",
```

Comment:
@shailesh896 shouldn't we use hyphen separator(i.e "/change-password") on the urls instead of underscore?